Mathlib/CategoryTheory/Comma/Over/Pullback.lean

```diff
@@ -156,14 +171,38 @@ def pullbackComp {X Y Z : C} (f : X ⟶ Y) (g : Y ⟶ Z) : pushout (f ≫ g) ≅
 instance pushoutIsLeftAdjoint {X Y : C} (f : X ⟶ Y) : (pushout f).IsLeftAdjoint  :=
   ⟨_, ⟨mapPushoutAdj f⟩⟩
 
+section HasInitial
+variable [HasInitial C]
+
 /-- If `X : C` is initial, then the under category of `X` is equivalent to `C`. -/
-def equivalenceOfIsInitial {C : Type*} [Category C] {X : C} (hX : IsInitial X) :
-    Under X ≌ C where
-  functor := Under.forget X
-  inverse := { obj Y := Under.mk (hX.to Y), map f := Under.homMk f }
-  unitIso := NatIso.ofComponents (fun Y ↦ Under.isoMk (Iso.refl _) (hX.hom_ext _ _))
-  counitIso := NatIso.ofComponents (fun _ ↦ Iso.refl _)
+@[simps!]
+noncomputable def forgetMapTerminal :
+    forget _ ≅ map (initial.to X) ⋙ (equivalenceOfIsInitial initialIsInitial).functor :=
+  NatIso.ofComponents fun X ↦ .refl _
```

Comment:
```suggestion
@[simps!]
noncomputable def forgetMapInitial :
    forget _ ≅ map (initial.to X) ⋙ (equivalenceOfIsInitial initialIsInitial).functor :=
  NatIso.ofComponents fun X ↦ .refl _
```

This should perhaps take an arbitrary initial object and a random morphism into it. Unless this breaks the defeq and that you care about it.